zerver/openapi/zulip.yaml

```diff
@@ -8994,6 +8994,20 @@ paths:
           schema:
             type: boolean
           example: true
+        - name: web_mark_read_on_scroll_policy
+          in: query
+          description: |
+            Whether or not to mark messages as read when the client scrolls through their
+            feed.
+
+            **Changes**: New in Zulip 7.0 (feature level 173).
+          schema:
+            type: integer
+            enum:
+              - 1
+              - 2
+              - 3
```

Comment:
The description should document what these numeric values mean. For an example, see `color_scheme` just below.

Reply:
Sounds good, will make the changes later.